tests/cli/functional/conftest.py

```diff
@@ -58,9 +149,12 @@ def test_registry():
 
 @pytest.fixture(scope='module')
 def test_registry_source_manager(mock_testerchain, test_registry):
-    return make_mock_registry_source_manager(blockchain=mock_testerchain,
-                                             test_registry=test_registry,
-                                             mock_backend=True)
+    real_inventory = make_mock_registry_source_manager(blockchain=mock_testerchain,
+                                                       test_registry=test_registry,
+                                                       mock_backend=True)
+    yield
+    # restore the original state
+    NetworksInventory.NETWORKS = real_inventory
```

Comment:
Perhaps both the `yield` and the restoration should be done in `make_mock_registry_source_manager`.

Reply:
:heavy_check_mark: